packages/docs/getting-started/running-a-validator.md

```diff
@@ -299,30 +308,42 @@ In order to be elected as a Validator, you will first need to register your grou
 Register your Validator Group:
 
 ```bash
+# On your local machine
 celocli validatorgroup:register --from $CELO_VALIDATOR_GROUP_ADDRESS --commission 0.1
 ```
 
 Register your Validator:
 
 ```bash
+# On your local machine
 celocli validator:register --from $CELO_VALIDATOR_ADDRESS --blsKey $CELO_VALIDATOR_BLS_PUBLIC_KEY --blsPop $CELO_VALIDATOR_BLS_SIGNATURE
 ```
 
+We don't want to use our account key for validation, so let's authorize the validating key:
+
+```bash
+# On your local machine
+celocli account:authorize --from $CELO_VALIDATOR_ADDRESS --role validator --pop $CELO_VALIDATOR_SIGNER_POP --signer $ATTESTATION_SIGNER_ADDRESS
+```
+
 Affiliate your Validator with your Validator Group. Note that you will not be a member of this group until the Validator Group accepts you:
 
 ```bash
+# On your local machine
 celocli validator:affiliate $CELO_VALIDATOR_GROUP_ADDRESS --from $CELO_VALIDATOR_ADDRESS
 ```
 
 Accept the affiliation:
 
 ```bash
+# On your local machine
 celocli validatorgroup:member --accept $CELO_VALIDATOR_ADDRESS --from $CELO_VALIDATOR_GROUP_ADDRESS
 ```
 
 Use both accounts to vote for your Validator Group:
 
 ```bash
+# On your local machine
```

Comment:
\# Note that because we have not authorized a vote signer, these transactions must be sent from the account keys. Since you're likely to need to place additional votes throughout the course of the stake-off, consider creating and authorizing vote signers for additional operational security.